### DISCUSSION TITLE
Options for mtz2cif for capturing Rfree set (FREER column) information

### DISCUSSION THREAD

**Discussion by glevans** [project-gemmi/gemmi#310] [Q&A]
Hi,

I am exploring Gemmi's options for conversion from mtz files to structure factor mmCIF files.

I am interested in converting contents of mtz file in 'FREER' column into both contents for:
 _refln.pdbx_r_free_flag
 _refln.status

I suspect this might be an option available for Gemmi mtz2cif using:
 --spec=FILE 

But I am not entirely sure how to indicate this in the 'spec' file.

It seems by default the binning captured in mtz file in 'FREER' column is being converted to mmCIF file _refln.status, with presumably default identifier (0?) of which intensities correspond to 'f' in the _refln.status (indicating the Rfree set).

It would be nice to capture all of the bins that have been set when the 'FREER' column is generated, as well as what Rfree set/bin has been selected.

Thank you for any feedback or insight that can be provided.

**Comment by wojdyr** [comment]
Hi Genevieve,

you could use the default spec as a starting point. `gemmi mtz2cif --print-spec` prints two specs, the first one is for merged files.
Then add a lines such as
```
? FREER I pdbx_r_free_flag
```
and use `gemmi mtz2cif --spec=filename ...`.

> It seems by default the binning captured in mtz file in 'FREER' column is being converted to mmCIF file _refln.status, with presumably default identifier (0?) of which intensities correspond to 'f' in the _refln.status (indicating the Rfree set).

Yes, unless the majority of reflections has 0 as a free flag. Then it's assumed that the PHENIX convention is used: 0 means working set and 1 – free set. Or you can specify the value explicitly with `--nfree=N`.
